docs/integrations/sources/postgres.md

```diff
@@ -291,6 +291,7 @@ According to Postgres [documentation](https://www.postgresql.org/docs/14/datatyp
 
 | Version | Date       | Pull Request                                             | Subject                                                                                                                                                                    |
 |---------|------------|----------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| 3.1.14  | 2023-10-16 | [31029](https://github.com/airbytehq/airbyte/pull/31029) | [Internal Refactor] : Remove strict-encrypt variants.                                                                                                                      |
```

Comment:
Is it really the case that this is an internal refactor? Strictly speaking (ha!) this connector now behaves differently when the env var `DEPLOYMENT_MODE=CLOUD` is set.